pontoon/contributors/static/js/settings.js

```diff
@@ -23,4 +23,19 @@ $(function() {
       }
     });
   });
+
+  // Show a warning when a user changes their email address.
+  var emailField = $('#id_email');
+  var warningContent = $('#email-warning');
+  var originalEmail = emailField.val();
+
+  emailField.on('keyup', function () {
+    var newEmail = emailField.val();
+    if (newEmail === originalEmail && warningContent.is(':visible')) {
+      warningContent.hide();
+    }
+    else if (newEmail !== originalEmail && !warningContent.is(':visible')) {
+      warningContent.show();
+    }
+  });
```

Comment:
I'm not a big fan of this. It doesn't work if you paste the email address using the mouse and the context menu for example. It also adds a lot of code.

Unrelatedly, there's a usability issue with this text - it gets overflown by the autocomplete box. We could move it to the right of the field, and possibly make it fit into two lines.

![screen shot 2018-01-15 at 18 28 18](https://user-images.githubusercontent.com/626716/34955079-35a9fc72-fa23-11e7-9d57-340186276fdf.png)

Reply:
I agree about the usability issue, and I'm going to fix it, but I don't understand your point about "a lot of code". That's only 10 lines, and I don't think it can be done with much less. The CSS-only solution had a different bunch of usability issues. Do you think we should do without that feature entirely instead?

Reply:
A lot of code compared to the CSS solution. What are the usability issues there?

Reply:
Quoting my commit message: 

> It was bad that the warning for a changed email would only appear on focus on the input. First it made it difficult to click the save button, as the layout was changed and thus the button moved. Second, it would show up even if the email was not changed.

I was going to consider going for a tooltip to the right of the input box instead of a red text below. What do you think?

Reply:
You could circumvent the layout change by toggling the `visibility` property instead of `display`. But that would still make the UI "blink".

Tooltip sounds good. Do you want to do something fancy? Helper text would also work:
![screen shot 2018-01-16 at 14 36 39](https://user-images.githubusercontent.com/626716/34991676-b872def6-faca-11e7-9b8d-124147a40a80.png)